csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
Makes sense. I think this change is fine if we verify it works with mono, monodevelop and xamaring (xbuild and mdtool).

And also that the nuget contains xmldoc and assemblies are signed - that's unlikely to break given the nature of this change, but still it's worth checking I think (https://github.com/grpc/grpc/pull/7102).